app/org/sagebionetworks/bridge/dynamodb/DynamoUserConsent2.java

```diff
@@ -97,6 +97,13 @@ public String getStudyKey() {
     public void setStudyKey(String studyKey) {
         this.studyKey = studyKey;
     }
+    @Override
+    public String getStudyIdentifier() {
+        return studyKey;
+    }
+    public void setStudyIdentifier(String studyIdentifier) {
+        this.studyKey = studyIdentifier;
+    }
```

Comment:
Minor: This will have the side effect of writing a duplicate field into the UserConsent2. Should we annotate these with DynamoDBIgnore? Alternatively, rename studyKey to studyIdentifier, but annotate with DynamoDBProperty to keep the name "studyKey"?